WordPressAuthenticator/Authenticator/WordPressAuthenticatorStyles.swift

```diff
@@ -80,6 +85,10 @@ public struct WordPressAuthenticatorStyle {
     ///
     public let prologueTitleColor: UIColor
 
+    /// Style: prologue top container child view controller
```

Comment:
I wonder if we can add more information here about what we can use this for. And what happens if it is `nil`?

Reply:
Yup, added in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/373/commits/7f6b0f6c8729d77bb85f133541a70af019f146a6